spec/models/mixins/virtual_total_mixin_spec.rb

```diff
@@ -15,6 +15,19 @@
       expect(model_with_children(2).total_vms).to eq(2)
     end
 
+    it "can bring back totals in primary query" do
+      m3 = model_with_children(3)
+      m1 = model_with_children(1)
+      m2 = model_with_children(2)
+      mc = m1.class
+      expect {
+        ms = mc.select(:id, mc.arel_attribute(:total_vms).as("total_vms"))
+               .order(:total_vms => :desc)
```

Comment:
thanks. just showing that you can sort by the column. but that is not relevant. removed